mars/conftest.py

```diff
@@ -141,21 +157,7 @@ def _new_integrated_test_session(_stop_isolation):
         try:
             yield sess
         finally:
-            try:
```

Comment:
This fragment of code stops unstoppable processes to prevent test hang at exit. If it causes ray to function abnormally, some conditions may be added instead of removing this code block directly.

Reply:
reverted.